test/Table_Tests/src/Common_Table_Operations/Conversion_Spec.enso

```diff
@@ -1,10 +1,12 @@
 from Standard.Base import all
 import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
 
-from Standard.Table import Value_Type
 import Standard.Table.Data.Type.Value_Type.Bits
-
+from Standard.Table import Value_Type
 from Standard.Table.Errors import Missing_Input_Columns, Conversion_Failure
+
+import Standard.Database.Data.Table.Table
```

Comment:
Let's make it clear that this is the DB table - otherwise it will be very easy to miss.
```suggestion
import Standard.Database.Data.Table.Table as Database_Table
```

Reply:
Done